src/Microsoft.Data.Sqlite/Properties/Microsoft.Data.Sqlite.rd.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
+  <Library>
+    <Type Name="Microsoft.Data.Sqlite.Interop.Sqlite3Handle" MarshalObject="Required All" />
+    <Type Name="Microsoft.Data.Sqlite.Interop.Sqlite3StmtHandle" MarshalObject="Required All" />
```

Comment:
What exactly does this tell it to do? How is it different from the default behavior?